imap_data_access/cli.py

```diff
@@ -33,6 +34,50 @@ def _download_parser(args: argparse.Namespace):
     print(f"Downloaded the file to: {output_path}")
 
 
+def _print_query_results_table(query_results):
```

Comment:
What you have here works just fine, and looks nice! I wonder if you could achieve a similar result with simpler/fewer lines of code by using a dataframe? Something along the lines of this: https://chat.openai.com/share/61f0f4e5-2ea9-482e-a424-86be1896e6d4

Reply:
I agree, this could definitely be done easier with external libraries 😂 . But, I was really striving for this package to have 0 dependencies and just be able to be installed/used as a script with no fuss.

Reply:
Yeah, I can see the benefit to that. This seems to work fine, so happy to go with it.

Reply:
We can have a summer student make this nicer later :)